internal/exec/describe_affected_utils.go

```diff
@@ -300,56 +300,56 @@ func executeDescribeAffected(
 		return nil, err
 	}
 
-	u.PrintInfoVerbose(verbose, fmt.Sprintf("\nGetting current working repo commit object..."))
+	u.LogInfo(cliConfig, fmt.Sprintf("\nGetting current working repo commit object..."))
 
 	localCommit, err := localRepo.CommitObject(localRepoHead.Hash())
 	if err != nil {
 		return nil, err
 	}
 
-	u.PrintInfoVerbose(verbose, fmt.Sprintf("Got current working repo commit object"))
-	u.PrintInfoVerbose(verbose, fmt.Sprintf("Getting current working repo commit tree..."))
+	u.LogInfo(cliConfig, fmt.Sprintf("Got current working repo commit object"))
+	u.LogInfo(cliConfig, fmt.Sprintf("Getting current working repo commit tree..."))
 
 	localTree, err := localCommit.Tree()
 	if err != nil {
 		return nil, err
 	}
 
-	u.PrintInfoVerbose(verbose, fmt.Sprintf("Got current working repo commit tree"))
-	u.PrintInfoVerbose(verbose, fmt.Sprintf("Getting remote repo commit object..."))
+	u.LogInfo(cliConfig, fmt.Sprintf("Got current working repo commit tree"))
+	u.LogInfo(cliConfig, fmt.Sprintf("Getting remote repo commit object..."))
 
 	remoteCommit, err := remoteRepo.CommitObject(remoteRepoHead.Hash())
 	if err != nil {
 		return nil, err
 	}
 
-	u.PrintInfoVerbose(verbose, fmt.Sprintf("Got remote repo commit object"))
-	u.PrintInfoVerbose(verbose, fmt.Sprintf("Getting remote repo commit tree..."))
+	u.LogInfo(cliConfig, fmt.Sprintf("Got remote repo commit object"))
+	u.LogInfo(cliConfig, fmt.Sprintf("Getting remote repo commit tree..."))
 
 	remoteTree, err := remoteCommit.Tree()
 	if err != nil {
 		return nil, err
 	}
 
-	u.PrintInfoVerbose(verbose, fmt.Sprintf("Got remote repo commit tree"))
-	u.PrintInfoVerbose(verbose, fmt.Sprintf("Finding diff between the current working branch and remote target branch ..."))
+	u.LogInfo(cliConfig, fmt.Sprintf("Got remote repo commit tree"))
+	u.LogInfo(cliConfig, fmt.Sprintf("Finding diff between the current working branch and remote target branch ..."))
 
 	// Find a slice of Patch objects with all the changes between the current working and remote trees
 	patch, err := localTree.Patch(remoteTree)
 	if err != nil {
 		return nil, err
 	}
 
-	u.PrintInfoVerbose(verbose, fmt.Sprintf("Found diff between the current working branch and remote target branch"))
-	u.PrintInfoVerbose(verbose, "\nChanged files:")
+	u.LogInfo(cliConfig, fmt.Sprintf("Found diff between the current working branch and remote target branch"))
+	u.LogInfo(cliConfig, "\nChanged files:")
```

Comment:
I suppose this should be info, since the user is running this command specifically to get this output.